3ds/SDL/SDL-1.2.15.patch

```diff
@@ -1698,27 +1731,6 @@ index 000000000..6ad575b20
 +	
 +	hidScanInput();
 +
-+	int i;
-+	SDL_keysym keysym;
-+	keysym.mod = KMOD_NONE;
-+
-+	for (i = 0; i < N3DS_NUMKEYS; i++) {
-+		keysym.scancode = i;
-+		keysym.sym = keymap[i];
-+
-+		if (hidKeysHeld() & (1 << i) && !keymem[i]) {
-+			keymem[i] = 1;
-+
-+			SDL_PrivateKeyboard (SDL_PRESSED, &keysym);
-+		}
-+
-+		if (!(hidKeysHeld() & (1 << i)) && keymem[i]) {
-+			keymem[i] = 0;
-+
-+			SDL_PrivateKeyboard (SDL_RELEASED, &keysym);
-+		}
-+	}
-+
 +	if (hidKeysHeld() & KEY_TOUCH) {
```

Comment:
Not super happy about it, but I guess it doesn't hurt, so I left the touchscreen mapping to the mouse API. Proper support for touchscreens was added to SDL2, so I guess this is better than nothing.